src/azure-cli/azure/cli/command_modules/network/tunnel.py

```diff
@@ -103,9 +104,10 @@ def _get_auth_token(self):
                 exp = CloudError(response)
             raise exp
 
+        self.ws_token = response_json["websocketToken"]
```

Comment:
It looks like this is only used once and that's through the return. Given that maybe we don't need to save it as a class variable.

Reply:
makes sense.